src/sentry/utils/sdk_crashes/sdk_crash_detection.py

```diff
@@ -8,10 +8,10 @@
 from sentry.eventstore.models import Event
 from sentry.issues.grouptype import GroupCategory
 from sentry.utils.safe import get_path, set_path
-from sentry.utils.sdk_crashes.cocoa_sdk_crash_detector import _cocoa_sdk_crash_detector_config
 from sentry.utils.sdk_crashes.event_stripper import strip_event_data
 from sentry.utils.sdk_crashes.sdk_crash_detection_config import SDKCrashDetectionConfig, SdkName
 from sentry.utils.sdk_crashes.sdk_crash_detector import SDKCrashDetector
+from sentry.utils.sdk_crashes.sdk_crash_detector_configs import _cocoa_sdk_crash_detector_config
```

Comment:
Might have missed this in a previous PR: Why are we importing a private function here? If `_cocoa_sdk_crash_detector_config` is intended as public, let's remove the `_` prefix.